apps/src/studio/Item.js

```diff
@@ -315,10 +340,14 @@ Item.prototype.removeElement = function() {
 };
 
 /**
- * Stop our animations
+ * Retrieve animation frame duration (frames per tick)
  */
-Item.prototype.stopAnimations = function() {
-  this.animation_.stopAnimator();
+Item.prototype.getAnimationFrameDuration = function () {
+  if (this.dir === Direction.NONE) {
+    return this.normalFrameDuration;
```

Comment:
When I changed the animate rate based on speed, it works well for walking/flying (where steps will move faster). But the idle animation shouldn't be speed up when not moving. So `Direction.NONE` implies "not moving", which essentially means `speed` is not relevant.